index.src.html

```diff
@@ -822,10 +830,10 @@ <h4 id="clear-dom">
      
       1.  An origin's WebSQL databases [[!WEBDATABASE]].
       2.  An origin's filesystems [[!file-system-api]]
-      3.  ChannelID
-      4.  Plugin data (e.g. <a href="https://wiki.mozilla.org/NPAPI:ClearSiteData">NPP_ClearSiteData</a>)
-      5.  Appcache.
-      6.  ISSUE: Moar?
+      3.  Plugin data (e.g. <a href="https://wiki.mozilla.org/NPAPI:ClearSiteData">NPP_ClearSiteData</a>),
+          except Flash [[!FLASH]] LSOs (see step #7 in [[#clear-cookies]]).
```

Comment:
Done. Flash is now included in both options. Given that the list for "storage" has a brainstorming character, I didn't elaborate on the scoping.

Related questions and notes:
- Do we want to rewrite the section so that it's more formal?
- Do we want to link a spec to app cache? IIUC it's deprecated, but while it's not removed, it still makes sense to list it and link to an older spec.
- Regarding your "Issue: MOAR?" - We currently list everything that the Chromium implementation deletes at the content layer. So there might be more, but you didn't forget anything obvious.